package.json

```diff
@@ -19,8 +21,8 @@
     "lint"
   ],
   "scripts": {
-    "start": "nodemon src/index.js",
-    "lint": "semistandard src/index.js",
+    "start": "nodemon src/server/index.js",
```

Comment:
I split up the route handler code (index.js) into its own files, so now there are multiple files on the server side so it seems more organized to keep the server files separate from the public files